third_party/move/move-compiler-v2/tests/reference-safety/v1-borrow-tests/mutate_with_borrowed_loc_struct_invalid.exp

```diff
@@ -0,0 +1,31 @@
+
+Diagnostics:
+error: cannot assign to borrowed local `x`
+  ┌─ tests/reference-safety/v1-borrow-tests/mutate_with_borrowed_loc_struct_invalid.move:7:9
+  │
+6 │         let y = &x;
+  │                 -- previous local borrow
+7 │         x = X { b: true };
+  │         ^^^^^^^^^^^^^^^^^ attempted to assign here
+8 │         move y;
+  │         ------ conflicting reference `y` used here
+
+error: cannot assign to borrowed local `s`
+   ┌─ tests/reference-safety/v1-borrow-tests/mutate_with_borrowed_loc_struct_invalid.move:16:9
+   │
+14 │         let y = &s;
+   │                 -- previous local borrow
+15 │         let z = &y.z;
+   │                 ---- used by field borrow
+16 │         s = S { z: 7 };
+   │         ^^^^^^^^^^^^^^ attempted to assign here
+17 │         z;
+   │         - conflicting reference `z` used here
+
+
+Diagnostics:
+error: local `x` of type `M::X` does not have the `drop` ability
+  ┌─ tests/reference-safety/v1-borrow-tests/mutate_with_borrowed_loc_struct_invalid.move:6:17
+  │
+6 │         let y = &x;
+  │                 ^^ still borrowed but will be implicitly dropped later since it is no longer used
```

Comment:
V1 also shows an erro about a dangling reference:
```
error[E07003]: invalid operation, could create dangling a reference
  ┌─ tests/move_check/translated_ir_tests/move/borrow_tests/mutate_with_borrowed_loc_struct_invalid.move:7:9
  │
6 │         let y = &x;
  │                 -- It is still being borrowed by this reference
7 │         x = X { b: true };
  │         ^ Invalid assignment of variable 'x'
```
File an issue to look into what happens if other errors are fixed.  Do we then catch this problem, or do we pass the program?